spec/models/orphan_spec.rb

```diff
@@ -464,23 +434,18 @@
                                                                     active_unsponsored_high_priority_orphan]
         end
 
-        specify '.active.currently_unsponsored should correctly return active unsponsored orphans only' do
-          expect(Orphan.active.currently_unsponsored.to_a).to match_array [active_unsponsored_orphan,
-                                                                 active_previously_sponsored_orphan,
-                                                                 active_previously_sponsored_high_priority_orphan,
-                                                                 active_unsponsored_high_priority_orphan]
-        end
-
         specify '.high_priority should correctly return high-priority orphans' do
           expect(Orphan.high_priority.to_a).to match_array [active_previously_sponsored_high_priority_orphan,
                                                             active_unsponsored_high_priority_orphan]
         end
 
         specify '.sort_by_eligibility should sort eligible orphans by sponsored_status, then priority' do
-          Orphan.sort_by_eligibility.to_a.each_with_index.map do |orphan, index|
-            expect([active_previously_sponsored_high_priority_orphan, active_previously_sponsored_orphan,
-                    active_unsponsored_high_priority_orphan, active_unsponsored_orphan][index]).to eq orphan
-          end
+          expect(Orphan.sort_by_eligibility).to eq [
+            active_previously_sponsored_high_priority_orphan,
+            active_previously_sponsored_orphan,
+            active_unsponsored_high_priority_orphan,
+            active_unsponsored_orphan
+          ]
```

Comment:
I had changed this from that because it was not failing when the same array elements were in a different order.  Not sure why.  So I picked each one out in turn and compared it